jessie/micro/Dockerfile

```diff
@@ -8,7 +8,9 @@ RUN apt-get update && apt-get install -y \
 		subversion \
 	&& rm -rf /var/lib/apt/lists/*
 
-RUN apt-get update && apt-get install -y \
+RUN apt-get update && apt-get install -y --no-install-recommends \
+		ca-certificates \
 		curl \
+		openssh-client \
```

Comment:
Shouldn't this one be in the above list, since it's really a SCM thing, not an HTTP(S)/FTP fetching tool?